base/rational.jl

```diff
@@ -90,8 +90,27 @@ promote_rule{T<:Integer,S<:AbstractFloat}(::Type{Rational{T}}, ::Type{S}) = prom
 
 widen{T}(::Type{Rational{T}}) = Rational{widen(T)}
 
+"""
+    rationalize([Type<:Integer,] x; tol::Real=eps(x))
```

Comment:
I think `T<:Integer` might be more consistent. And should mention that if not provided, `T` defaults to `Int`.